project/AkkaBuild.scala

```diff
@@ -136,7 +136,7 @@ object AkkaBuild extends Build {
     id = "akka-cluster-sharding",
     base = file("akka-cluster-sharding"),
     dependencies = Seq(cluster % "compile->compile;test->test;multi-jvm->multi-jvm",
-        persistence % "compile;test->provided", distributedData % "compile;test->provided", clusterTools)
+        persistence % "compile;test->provided", distributedData % "provided;test", clusterTools)
```

Comment:
Now it actually becomes scope "test" in the pom.xml. It should be "provided" in the pom.xml, but I don't know how to do that. If I only use "provided" here it works, but then we can't run tests. Scope "test" is alright in the pom, so if we don't have any better suggestions we'll use that.